docs/components/modeler/desktop-modeler/element-templates/configuring-templates.md

```diff
@@ -5,7 +5,7 @@ title: Configuring templates
 
 Element templates are defined as [JSON files](../defining-templates). There are two ways to integrate them into the Camunda Modeler:
 
-1. **Retrieve from Cawemo:** use the [Camunda Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well). Locally the synced templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](../../search-paths#user-data-directory). You should not manually change the `config.json` unless you know what you're doing.
+1. **Retrieve from Cawemo:** use the [Camunda Platform 8 Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) to integrate the Camunda Modeler with [Cawemo](https://cawemo.com/). The Modeler can then retrieve templates from catalog projects setup in Cawemo. Note that the [Cloud Connect plugin](https://docs.camunda.org/cawemo/latest/technical-guide/integrations/modeler/) will keep Cawemo and your local Camunda Modeler installation in sync (e.g., deleting a template in Cawemo will delete it locally as well). Locally the synced templates will be stored in the `config.json` file in your [`{USER_DATA_DIRECTORY}`](../../search-paths#user-data-directory). You should not manually change the `config.json` unless you know what you're doing.
```

Comment:
The Plugin will not be renamed

Reply:
And moreover, the plugin is for Camunda Platform 7, not 8.

Reply:
Do you want me to be more clear that this is for C7? Usually, we badge the page or section of "Camunda Platform 7 only".